Bottom: 8df88534cd4cfb318f494e8100bdd8ede684b9d0
Top:    c40c7b52173467542d24b59755d8f1673efce286
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-02 17:02:56 +0800

Refresh of autofs-5.0.7-fix-fix-wildcard-multi-map-regression.patch

---

diff --git a/modules/lookup_file.c b/modules/lookup_file.c
index e9697e8..4e4317f 100644
--- a/modules/lookup_file.c
+++ b/modules/lookup_file.c
@@ -1084,7 +1084,6 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 		}
 	}
 
-	error(LOGOPT_ANY, "take cache writelock");
 	/*
 	 * We can't take the writelock for direct mounts. If we're
 	 * starting up or trying to re-connect to an existing direct
@@ -1096,7 +1095,6 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 		cache_writelock(mc);
 	else
 		cache_readlock(mc);
-	error(LOGOPT_ANY, "got cache writelock");
 do_cache_lookup:
 	me = cache_lookup(mc, key);
 	/*
